tests/conftest.py

```diff
@@ -136,17 +141,17 @@ def runcmd(self, cmd):
         try:
             out = subprocess.check_output("ip netns exec %s %s" % (self.nsname, cmd), stderr=subprocess.STDOUT, shell=True)
         except subprocess.CalledProcessError as e:
-            print "------rc={} for cmd: {}------".format(e.returncode, e.cmd)
-            print e.output.rstrip()
-            print "------"
+            print(("------rc={} for cmd: {}------".format(e.returncode, e.cmd)))
+            print((e.output.rstrip()))
```

Comment:
Any idea why these extra parens are here?

Reply:
it was added by 2to3. removed extra parens